.pre-commit-config.yaml

```
@@ -0,0 +1,15 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
```

```suggestion
      - id: end-of-file-fixer
      - id: remove-tabs
```

I'm considering adding 1 more hook: 
```
remove-tabs - Replace tabs by whitespaces before committing
```
this one for cadence is a bit drastic but at least we will be sure that there is no identation issue.
All my cadence files are like this. But I know you use tabs. What do you think about that one?